addons/cookoff/functions/fnc_handleDamage.sqf

```diff
@@ -21,6 +21,9 @@ _thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIn
 // it's already dead, who cares?
 if (damage _vehicle >= 1) exitWith {};
 
+// Check for players and exit if none found and the enable for players only setting is true
+if (_vehicle getVariable [QGVAR(enable), GVAR(enable)] isEqualTo 2 && {fullCrew [_vehicle, "", false] findIf {isPlayer (_x select 0)} == -1}) exitWith {};
```

Comment:
I think it is weird that this variable is checked in the event handler added in postInit, but then also again in the called function. This can probably be simplified. To avoid copying the line over and over, I would move all checks in the function.

Reply:
It's checked in the EH because at this point it's deciding if it should be added at all and was updated to the `in` check to support people who've set the variable manually as per @mharis001's suggestion.

Surely we want to keep this so the EH isn't added if people have the setting disabled? The function call is checking if it's just enabled or enabled for players only so is a slightly different check.

If it's all moved in to the function, it'll add the EH even when the setting is disabled. I'm likely misunderstanding, grab me on Slack.

Reply:
Moved check to be in fnc. Only checking if disabled because if it's 1/true it'll run and if it's 2 it has it's logic check already built in.